src/pseudopeople/interface.py

```diff
@@ -367,9 +367,9 @@ def generate_women_infants_and_children(
     :param seed: An integer seed for randomness. Defaults to 0.
     :param config: An optional override to the default configuration. Can be a path
         to a configuration YAML file or a dictionary.
-    :param year: The year (format YYYY) to include in the dataset. Will raise a
-        ValueError if there is no data for this year. If None is provided,
-        data from all years are included in the dataset.
+    :param year: The year in which WIC benefits were received (format YYYY, e.g., 2036). Will raise a
```

Comment:
I'm a bit worried about duplicating and/or contradicting the Datasets page, which says: "This is a yearly file of information about all simulants enrolled in the program as of the end of that year. For the final year available, 2041, the file includes those enrolled as of May 1st, because this is the end of our simulated timespan." Perhaps we should consider including all that info here, directly referencing the Datasets page for more info, or something else I'm not thinking of?

Arguably not incompatible with what you've written here, just more detailed.

It's more minor, but the Datasets page is more detailed with respect to the household surveys, as well.

Reply:
I basically copied the descriptions of "year" and "tax_year" from the descriptions of those columns on the Datasets page. Do you think we need to edit those descriptions as well?

Reply:
I think the past tense issue is specific to the documentation of these arguments, and does not affect the documentation of the columns themselves. The fact that these correspond is more of a coincidence (see: survey_date and year for the survey datasets and functions respectively).

I think we might do better to adapt these parameter descriptions from the portion of the Datasets page I quoted, the part above each table where we say how frequently the file is produced, rather than the column descriptions. Brevity is a concern, though I do tend to think of docstrings as being at least as "in the weeds" as higher-level documentation.

Reply:
I edited all the docstrings to use present and/or future tense. For WIC specifically (and also probably for SSA and 1040, at least), I think we should include some of the description from the Dataset page at the top of the docstrings, but I haven't done that yet so we can focus on getting this merged. For now, I briefly noted the peculiarity of Dec 31 vs. May 1 for WIC in the description of the year parameter.